src/navigation/Nav.jsx

```diff
@@ -262,7 +262,6 @@ export class Nav extends React.Component {
 			isSearchEnabled &&
 				!media.isAtMediumUp && {
 					shrink: true,
-					icon: search.icon,
```

Comment:
I would keep this icon for backward compatibility. Maybe do something like this instead: 
```icon: !isNewNavActive ? search.icon : undefined```